doc/install.rst

```diff
@@ -15,15 +15,13 @@ The following packages are required to use Elephant:
     * numpy_ >= 1.8.2
     * scipy_ >= 0.14.0
     * quantities_ >= 0.10.1
-    * neo_ == 0.4.0
-
-.. note:: at the time of writing Neo 0.4.0 has not been released. You should therefore use `this snapshot`_.
+    * neo_ == 0.5.0
```

Comment:
I suggest making this `>=`, so when we release Neo 0.5.1, etc., we don't need to change this.